packages/core/lib/segments/attributes/subsegment.d.ts

```diff
@@ -61,6 +61,8 @@ declare class Subsegment {
   toString(): string;
 
   toJSON(): { [key: string]: any };
+
+  serialize(object?: Subsegment): string;
```

Comment:
```suggestion
  serialize(subsegment?: Subsegment): string;
```

Better naming would be `subsegment` instead of `object`, which follows the naming convention used in the methods above as well